### ISSUE TITLE
Regression in 2.8.0

### ISSUE THREAD

**Issue by daffl** [ThomasAribart/json-schema-to-ts#133]
Initially reported in https://github.com/feathersjs/feathers/issues/3168, the 2.8.0 release produces the following error in all Feathers apps that are using `json-schema-to-ts`:

```
node_modules/json-schema-to-ts/lib/types/index.d.ts:16:314 - error TS2590: Expression produces a union type that is too complex to represent.
```

I didn't have much of a chance to look into it and since there isn't a Changelog I can't really tell what changed but wanted to flag the problem it since it comes up a lot now.

**Comment by daffl** [comment]
I just noticed that this is probably the same as https://github.com/ThomasAribart/json-schema-to-ts/issues/132

**Comment by ThomasAribart** [comment]
Hey ! Sorry I was on holiday but will put this in high prio !

Will try to fix this this week-end. Do you have an example of failing schema ? Or does the error just suddenly happens ?

**Comment by ThomasAribart** [comment]
Actually, closing this as it is a duplicate of https://github.com/ThomasAribart/json-schema-to-ts/issues/132

**Comment by ThomasAribart** [comment]
Should be fixed in 2.8.2
